### PR TITLE
Correct minor tutorial typos

### PR DESCRIPTION
**Pull request by loganthomas**
I recently watched Anderson Banihirwe's "Xarray Tutorial | xarray fundamentals" on YouTube ([here](https://www.youtube.com/watch?v=a339Q5F48UQ)). This PR attempts to correct a few *very minor* typos that I found while following along in the notebooks.

While it's not a typo fix, I also added a `misc` section to the `.gitignore` file to include `.DS_Store` and `.vscode/` so that those will not get pushed accidentally to the repo.

Happy to change anything that might be problematic.

Thank you for the great resources for Xarray. I'm really enjoying the tutorials so far!

### CODE REVIEW

**Comment by loganthomas** [comment]
Updated `overview/xarray-in-45-min.ipynb` to solve failing check in 8ef9c4a

File: workshops/online-tutorial-series/01_xarray_fundamentals.ipynb [hunk 1]
```diff
@@ -236,7 +236,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Extract the sst variable/datarray\n",
+    "# Extract the sst variable/data array\n",
```
Comment (dcherian) [inline workshops/online-tutorial-series/01_xarray_fundamentals.ipynb:239]:
```suggestion
    "# Extract the sst Variable/DataArray\n",
```

File: workshops/online-tutorial-series/01_xarray_fundamentals.ipynb [hunk 2]
```diff
@@ -260,7 +260,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# dataarray/variable dimensions\n",
+    "# data array/variable dimensions\n",
```
Comment (dcherian) [inline workshops/online-tutorial-series/01_xarray_fundamentals.ipynb:263]:
```suggestion
    "# DataArray/Variable dimensions\n",
```

File: workshops/online-tutorial-series/01_xarray_fundamentals.ipynb [hunk 3]
```diff
@@ -270,7 +270,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# datarray/variable coordinates\n",
+    "# data array/variable coordinates\n",
```
Comment (dcherian) [inline workshops/online-tutorial-series/01_xarray_fundamentals.ipynb:273]:
```suggestion
    "# DataArray/Variable coordinates\n",
```

File: workshops/online-tutorial-series/01_xarray_fundamentals.ipynb [hunk 4]
```diff
@@ -280,7 +280,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# dataarray/variable attributes\n",
+    "# data array/variable attributes\n",
```
Comment (dcherian) [inline workshops/online-tutorial-series/01_xarray_fundamentals.ipynb:283]:
```suggestion
    "# DataArray/Variable attributes\n",
```

File: workshops/online-tutorial-series/01_xarray_fundamentals.ipynb [hunk 5]
```diff
@@ -389,7 +389,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Set some arbitrary attribute on a data variable/datarray\n",
+    "# Set some arbitrary attribute on a data variable/data array\n",
```
Comment (dcherian) [inline workshops/online-tutorial-series/01_xarray_fundamentals.ipynb:392]:
```suggestion
    "# Set some arbitrary attribute on a data Variable/DataArray\n",
```

**Review by dcherian — COMMENTED** [review]
Thanks! DataArray is the proper term so I added capital letters to make it clear

**Pull request merged**